### PR TITLE
[MERGE][CLN] delivery: cleanup pass before incoming task(s)

### PR DESCRIPTION
**Pull request by vava-odoo**
- remove utf-8 encoding header in python scripts
- add missing odoo license and xml headers
- alphabetical order for imports
- rename files according to model names (+ split if needed)
- remove unused variables in tests

### CODE REVIEW

**Review by Feyensv — COMMENTED** [review]
`class StockReturnPicking(models.TransientModel):` could be moved from stock_picking.py to a dedicated file in wizard folder ;)

File: addons/delivery/models/delivery_carrier.py [hunk 1]
```diff
@@ -279,8 +294,6 @@ def _get_default_custom_package_code(self):
     # Fixed price shipping, aka a very simple provider #
     # ------------------------------------------------ #
 
-    fixed_price = fields.Float(compute='_compute_fixed_price', inverse='_set_product_fixed_price', store=True, string='Fixed Price')
-
```
Comment (Feyensv) [inline addons/delivery/models/delivery_carrier.py:283]: avoid moving fields/methods around for now (except the ones where you remove the file ofc)

File: addons/delivery/models/delivery_carrier.py [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
 import psycopg2
 import re
 
-from odoo import api, fields, models, registry, SUPERUSER_ID, _
+from odoo import _, api, fields, models, registry, SUPERUSER_ID
+from odoo.exceptions import UserError, ValidationError
 from odoo.tools.float_utils import float_round
-from odoo.exceptions import UserError
+from odoo.tools.safe_eval import safe_eval
```
Comment (Feyensv) [inline addons/delivery/models/delivery_carrier.py:9]:
```suggestion
from odoo.tools import float_round, safe_eval
```

File: addons/delivery/models/stock_move.py [hunk 3]
```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from odoo import api, fields, models
```
Comment (Feyensv) [inline addons/delivery/models/stock_move.py:3]: not motivated to sort the imports in stock_* files ? :D (or another reason ?)

**Comment by AntoineVDV** [comment]
@robodoo override=ci/security
Reason: moved files without code changes
Link to ci/security build (master): https://runbot.odoo.com/runbot/build/23508823

**Comment by Feyensv** [comment]
@robodoo r+ rebase-merge

**Pull request closed without merging**